examples/label_export/text.ipynb

```diff
@@ -2,22 +2,17 @@
   "cells": [
```

Comment:
<p>do we want to show this output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>